python/requirements.txt

```diff
@@ -78,7 +78,7 @@ pexpect
 Pillow; platform_system != "Windows"
 pygments
 pyspark==3.1.2
-pytest==5.4.3
+pytest<7.1
```

Comment:
do you think it's better to ping pytest==7.0.1

Reply:
Done.